lib/internal/watch_mode/files_watcher.js

```diff
@@ -97,8 +97,11 @@ class FilesWatcher extends EventEmitter {
       return;
     }
     const watcher = watch(path, { recursive, signal: this.#signal });
-    watcher.on('change', (eventType, fileName) => this
-      .#onChange(recursive ? resolve(path, fileName ?? '') : path));
+    watcher.on('change', (eventType, fileName) => {
+      // `fileName` can be `null` if it cannot be determined. See
+      // https://github.com/nodejs/node/pull/49891#issuecomment-1744673430.
```

Comment:
I think this comment should also be in the documentation: https://nodejs.org/docs/latest/api/fs.html#fswatchfilename-options-listener

At least to explain why they could receive empty string.

Reply:
It is already documented. See https://nodejs.org/docs/latest/api/fs.html#filename-argument.

Reply:
Oh, nice!